src/lib/Metadata/Metadata.js

```diff
@@ -0,0 +1,129 @@
+/*!
+ * Copyright(c) ConsenSys Software Inc.
+ * Copyright(c) https://consensys.net/
+ * MIT Licensed
+ */
+
+import { readFileSync } from 'fs';
+
+import IPFS from '../../services/ipfsService.js';
+import { ERROR_LOG, errorLogger } from '../error/handler.js';
+
+export default class Metadata {
+  constructor({ projectId, projectSecret, ipfsUrl = 'ipfs.infura.io:5001' }) {
```

Comment:
maybe from constants

Reply:
To me, we shouldn't have it as a parameter :)